### PR TITLE
Merge history tables and refactor HistoryService

### PR DESCRIPTION
**Pull request by sprucely**
Also tweaked some controller actions to be a little more consistent WRT admin overrides.

### CODE REVIEW

File: Hippo.Web/Controllers/AccountController.cs
Comment (jSylvestre) [inline Hippo.Web/Controllers/AccountController.cs:236]: This can't be hit now?
Do we need/want to allow them to update their SSH before it is approved?

**Review by jSylvestre — APPROVED** [review]
Didn't test, but except for my one comment, looks good to me.

File: Hippo.Web/Controllers/AccountController.cs
Comment (sprucely) [reply Hippo.Web/Controllers/AccountController.cs:236]: Oops. Forgot to change that above condition.

I believe the intent is that, if an account has already been approved, a user should be able to update their ssh without requiring further approval.

**Review by laholstege — APPROVED** [review]
didnt see anything beyond what jason pointed out 👍

**Pull request merged**
